modules/apps/translation/translation-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Translation API
 Bundle-SymbolicName: com.liferay.translation.api
-Bundle-Version: 5.5.0
+Bundle-Version: 6.0.0
```

Comment:
Hi @adolfopa 

this is a breaking change... so we should stop it till the release?